app/models/admin/user_mentee_application_cohort.rb

```diff
@@ -9,7 +9,7 @@
 #  created_at        :datetime         not null
 #  updated_at        :datetime         not null
 #
-class UserMenteeApplicationCohort < ApplicationRecord
+class Admin::UserMenteeApplicationCohort < ApplicationRecord
```

Comment:
So we shouldn't be changing the model namespace. This should remain `UserMenteeApplicationCohort` still.